File: docs-examples/example-java/src/test/java/io/micronaut/configuration/rabbitmq/docs/consumer/custom/annotation/DeliveryTag.java
package io.micronaut.configuration.rabbitmq.docs.consumer.custom.annotation;

// tag::imports[]
import io.micronaut.core.bind.annotation.Bindable;

import java.lang.annotation.*;
// end::imports[]

// tag::clazz[]
@Documented
@Retention(RetentionPolicy.RUNTIME)
@Target({ElementType.PARAMETER})
@Bindable // <1>
public @interface DeliveryTag {
}
// end::clazz[]

File: docs-examples/example-kotlin/src/test/kotlin/io/micronaut/configuration/rabbitmq/docs/consumer/custom/annotation/DeliveryTag.kt
package io.micronaut.configuration.rabbitmq.docs.consumer.custom.annotation

// tag::imports[]
import io.micronaut.core.bind.annotation.Bindable
import java.lang.annotation.Documented
// end::imports[]

// tag::clazz[]
@MustBeDocumented
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.VALUE_PARAMETER)
@Bindable // <1>
annotation class DeliveryTag
// end::clazz[]